hello-world/2025_03_08_international_womens_day.md

```diff
@@ -0,0 +1,48 @@
+# A tribute to two women who enabled my career
+
+## My great-grandmother
+My great-grandmother (the mother of the mother of my mother) is without any
+doubt someone who inspired my career, even if she did not live to see the result.
+
+Born in april 1895, she was the only relative I ever knew who was born in the
+19th century. Se was the youngest of thirteen children wo became a kindergarden
```

Comment:
S**h**e was the youngest of thirteen children who became a kindergar**t**en
teacher until she married (she was fired as result, which was ~common~ law back
in the day in the Netherlands, and not only there).

"Common law" is law system based on previous court rulings, like used in England. The Netherlands has a system based on civil law. I think you meant to say that firing a female teacher was common after she got married. In that time there was a law that forbid married women younger than 45 to work for the goverment. But IANAL and there are more details than I, and you I guess, care to explain in this story.